src/plugins/explore/public/application/utils/hooks/use_saved_explore.ts

```diff
@@ -75,6 +78,8 @@ export const useSavedExplore = (exploreIdFromUrl?: string) => {
           savedExploreObject.id,
           { type: 'explore' }
         );
+
+        dispatch(executeQueries({ services }));
```

Comment:
Does this action trigger search? If so, I'm a bit concern about it, because `useSavedExplore` is supposed to be reusable and currently already used in multiple component.

Reply:
I will remove this for now.